packages/manager/apps/cloud/client/app/cloud/project/compute/infrastructure/translations/Messages_fr_FR.json

```diff
@@ -158,7 +158,7 @@
   "cpci_volume_number_detach_other": "{{nbVolume}} disques dur détachés",
   "cpci_volume_add": "Ajouter un disque",
   "cpci_volume_attach": "Attacher à un serveur",
-  "cpci_volume_attach_error": "Vous ne possédez actuellement aucune instance compatible à laquelle attacher ce volume.",
+  "cpci_volume_attach_error": "Aucune instance n’est disponible dans la région de votre volume.",
```

Comment:
As seen together, this file isn't related to the Public Cloud application where changes are expected, so it could be removed from the diff.